src/SFA.DAS.EmployerIncentives.Web.SystemAcceptanceTests/Features/Application/ReadyToEnterBankDetails.feature

```diff
@@ -0,0 +1,24 @@
+﻿@employerIncentivesApi
+Feature: ReadyToEnterBankDetails
```

Comment:
Good Gherkin!

Maybe replace "redirected" (which is a bit techy) with "requested" and drop the reference to "page"?

Reply:
That's updated